homeassistant/components/tuya/translations/en.json

```diff
@@ -33,6 +33,20 @@
                     "2": "On"
                 }
             },
+            "blanket_level": {
+                "state": {
+                  "level_1": "Low",
+                  "level_2": "Level 2",
```

Comment:
Why would Level 1 be "Low", and not just "Level 1" ?

Reply:
I was mimicking the tuya Smart Life app interface for this device.  It uses an "L" and "H" for the heat settings.  I'm not set on the idea and can put it back to match the actual codes.

![Screenshot_20230122-103258](https://user-images.githubusercontent.com/106625807/214458674-8623e136-5360-43fd-adb0-51bc61b8b877.png)
![Screenshot_20230122-103358](https://user-images.githubusercontent.com/106625807/214458682-4f873983-24dc-4b11-b985-1d455d672c65.png)